eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/metric/ClusteringMetricCompute.java

```diff
@@ -31,6 +35,7 @@ public String getIdentifier() {
         try {
             JSONObject identifier = new JSONObject();
             identifier.put("assembly", assembly);
+            identifier.put("projects", projects);
```

Comment:
I think that what I wanted but does that now mean that every clustering metrics will have to report on project? Does it get ignored if it is `null` ?

Reply:
It'll have an empty list as part of the identifier if no projects are provided.